wiki/History_of_osu!/April_Fools/en.md

```diff
@@ -0,0 +1,324 @@
+4. [osu! forum comment (Resolved Issues) [Trosk-] - "[resolved] [confirmed] Regarding osu!supporter/Auto mod"](https://osu.ppy.sh/community/forums/topics/437902?start=5006190)
+
+---
+
+## 2017
+
+As announced in a [tweet by peppy](https://twitter.com/ppy/status/848021525663842304 "Twitter"), there was no April Fools' day joke for osu! in 2017 due to osu!lazer development.
+
+---
+
+## 2018
+
+### Rotating osu! cookie
+
+On 1 April 2018 the [osu! cookie](/wiki/Glossary/Cookie) on the main menu screen would slowly rotate clockwise as time passed, and the cookie in the song selection screen would slowly rotate anticlockwise. Hovering over these cookies would enlarge them just as usual, but would also cause them to rotate faster \[1,2,3,4,5,6\].
+
+#### References
+
+1. [Reddit post (r/osugame) [u/[deleted]] - "New April Fools Update now has a rotating osu! Logo"](https://www.reddit.com/r/osugame/comments/88kq23/new_april_fools_update_now_has_a_rotating_osu_logo/?st=JFGBCI5G&sh=49cf8677 "Reddit")
+2. [Reddit post (r/osugame) [u/hi_im_marc] - "April Fools Patch Is Out Get Ready To Get BAMBOOZLED!!!1"](https://www.reddit.com/r/osugame/comments/88kbit/april_fools_patch_is_out_get_ready_to_get/ "Reddit")
+3. [Reddit post (r/osugame) [u/AdriaLOL] - "haha, nice april fools peppy XD"](https://www.reddit.com/r/osugame/comments/88qlwk/haha_nice_april_fools_peppy_xd/ "Reddit")
+4. [osu! forum thread (General Discussion) [Aochie] - "The osu! logo is moving?"](https://osu.ppy.sh/community/forums/topics/724377)
+5. [osu! forum thread (Resolved Issues [Jreen] - "[resolved] Osu! Logo Sideways?"](https://osu.ppy.sh/community/forums/topics/724094)
+6. [osu! wiki article - "History of osu! 2018"](/wiki/History_of_osu!/2018#april)
+
+---
+
+## 2019
+
+### Sneezing girl sound effect (2019)
+
+For April Fools' day 2019 there was approximately a 1 in 20 chance of hearing the sound effect of a high-pitched girl sneezing upon opening a [beatmap](/wiki/Beatmap) \[1,2,3\].
+
+#### References
+
+1. [Reddit post (r/osugame) [u/jivko500] - "The April Fools joke in osu"](https://www.reddit.com/r/osugame/comments/b83pnl/the_april_fools_joke_in_osu/ "Reddit")
+2. [Reddit post (r/osugame) [u/anoymaly2152] - "Bless you, Pippi."](https://www.reddit.com/r/osugame/comments/b848ro/bless_you_pippi/ "Reddit")
+3. [osu! forum thread (Off-Topic) [Brainage] - "No April Fools in the changelog?"](https://osu.ppy.sh/community/forums/topics/888939)
+
+---
+
+## 2020
+
+### MillhioreF joins the Featured Artists
+
+[MillhioreF](https://osu.ppy.sh/users/941094)—a long-time osu! moderator, developer and [Easy mod](/wiki/Game_modifier/Easy) player—was announced in [a newspost](https://osu.ppy.sh/home/news/2020-04-01-new-featured-artist-millhioref) to have "joined" the list of [Featured Artists](/wiki/Featured_Artists) as "Millhiore Firianno Biscotti" on 1 April 2020 with a selection of five songs to boot \[1\]:
```

Comment:
You can also add a detail about millhi's featured artist ID, which was 727, though the page no longer exists.

![](https://zeu.s-ul.eu/awYdE8En.png)

Reply:
Ehhh. I typically enjoy added bits of trivia but this is just a bit too trivial imo. If anyone is really in dire need of this info they'll know how to find it